packages/Calendar/Microsoft.Bot.Component.CalendarSkill.csproj

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should this rename to Microsoft.Bot.Component.Calendar?

Reply:
Stale .csproj file - I guess this is what happens when I rename! Removed.